### ISSUE TITLE
POP time profile plot looks weird with 2 different outputs

### ISSUE THREAD

**Issue by Yuri05** [Open-Systems-Pharmacology/OSPSuite.ReportingEngine#208]
When >1 output curves are defined, time profile plots look strange (s. below).
In this example [workflow7.zip](https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/files/4695061/workflow7.zip) 2 output curves are defined:
<details><summary>SimModelSet</summary>

```
outPlasma  <- Output$new(path = 'Organism|PeripheralVenousBlood|Rifampicin|Plasma (Peripheral Venous Blood)', 
                         displayName = 'Rifampicin', 
                         displayUnit = 'µg/ml',
                         dataFilter = 'CMT==1',
                         dataDisplayName = 'Study data 9999')

outUrine <- Output$new(path = 'Organism|Kidney|Urine|Rifampicin|Fraction excreted',
                       displayName = 'excreted to urine',
                       displayUnit = '%')

popModelSet <- PopulationSimulationSet$new(simulationSetName = 'PO450mg',
                                  simulationFile = simFilepkml,
                                  populationFile = popFileCsv,
                                  outputs = c(outPlasma,outUrine),
                                  observedDataFile = dataFile,
                                  observedMetaDataFile = dictionaryFile)
```
</details>

Also the captions are wrong: both plots show y-axis caption "Fraction [%]"

One reason for both problems could be that 2 curves in this examples **have different dimensions** and must be put on 2 different axes.

![PO450mg-timeProfile](https://user-images.githubusercontent.com/25061876/83133624-6faf4800-a0e3-11ea-9038-d38a1a1b185a.png)![PO450mg-timeProfileLog](https://user-images.githubusercontent.com/25061876/83133639-750c9280-a0e3-11ea-8cbf-7ffec0a88b1e.png)

If I keep only 1 Curve in the Simulation Set, it looks good:
<details><summary>SimModelSet</summary>

```
popModelSet <- PopulationSimulationSet$new(simulationSetName = 'PO450mg',
                                  simulationFile = simFilepkml,
                                  populationFile = popFileCsv,
                                  outputs = c(outPlasma),
                                  observedDataFile = dataFile,
                                  observedMetaDataFile = dictionaryFile)
```
</details>

![PO450mg-timeProfile](https://user-images.githubusercontent.com/25061876/83134343-81451f80-a0e4-11ea-9a75-8b2e1b0b7a2c.png) ![PO450mg-timeProfileLog](https://user-images.githubusercontent.com/25061876/83134371-8a35f100-a0e4-11ea-8846-f4e041b9fef9.png)

**Comment by Yuri05** [comment]
Also some warnings are shown during workflow execution:
<details><summary>Warnings</summary>

```
[1] "28/05/2020 - 12:50:57 : Plot Goodness of Fit task in Alpha Testing for Rifampin po, 450 mg, Bioequivalence - PopSim"
Scale for 'y' is already present. Adding another scale for 'y', which will replace the existing scale.
Scale for 'y' is already present. Adding another scale for 'y', which will replace the existing scale.
Scale for 'x' is already present. Adding another scale for 'x', which will replace the existing scale.
Scale for 'y' is already present. Adding another scale for 'y', which will replace the existing scale.
Scale for 'y' is already present. Adding another scale for 'y', which will replace the existing scale.
[1] "28/05/2020 - 12:51:03 : Report './Output_Workflow7/Report.md' was initialized successfully."
Executing: pandoc --toc --self-contained --wrap=none -t gfm -o "./Output_Workflow7/Report.md" "./Output_Workflow7/Report.md"
Warnings:
1: Transformation introduced infinite values in continuous y-axis 
2: Transformation introduced infinite values in continuous y-axis 
3: Transformation introduced infinite values in continuous y-axis 
4: Transformation introduced infinite values in continuous y-axis 
5: Transformation introduced infinite values in continuous y-axis 
6: Transformation introduced infinite values in continuous y-axis 
7: Transformation introduced infinite values in continuous y-axis 
8: Transformation introduced infinite values in continuous y-axis 
9: Transformation introduced infinite values in continuous y-axis 
10: Transformation introduced infinite values in continuous y-axis 
```
</details>

**Comment by pchelle** [comment]
PR #211 is only partly fixing this issue because plotting the output(s) with different units on the second y-axis will be challenging.
Unlike `Matlab`, the second axis and its data are not independent from the first axis in `ggplot2`.
It means that we will need `tlf` to handle and perform a mapping/scaling of the data and the axis in the background every time this axis is requested. (I just wrote the issue for that)

As discussed during our last meeting, I am currently generating different plots when I found different units in outputs.
